pep-0570.rst

```diff
@@ -441,7 +490,7 @@ A draft of the proposed grammar specification is::
 It would be added to the actual ``typedargslist`` and ``varargslist``, but for
 more relaxed discussion it is presented as ``new_typedargslist`` and
 ``new_varargslist``. Note that using a construction with two new rules
-(``new_varargslist`` and ``new_varargslist``) is not possible with the current
+(``new_typedargslist`` and ``new_varargslist``) is not possible with the current
 parser as a rule is not LL(1). This is the reason the rule needs to be
 included in the existing ``typedargslist`` and ``varargslist`` (in the same way
 keyword-only arguments were introduced).
```

Comment:
That's a lot of words. IMO just showing the entire updated definitions for these two rules would be better.

Reply:
I removed the wording here, updated a simplified view of the change, and added the full rules. Hoping that clarifies.